tests/rule_based_profiler/data_assistant/test_volume_data_assistant.py

```diff
@@ -2581,59 +2653,56 @@ def test_execution_time_within_proper_bounds(
     assert data_assistant_result.execution_time > 0.0
 
 
-def test_volume_data_assistant_plot_descriptive(
-    quentin_columnar_table_multi_batch_data_context,
```

Comment:
more for my curiosity here, but what was the reason this was changed from `quentin` to `bobby`?

Reply:
No reason, I just created the new tests and then deleted the old ones. The diffs make it look like I changed them, but it is only an illusion.